addons/account/models/account.py

```diff
@@ -972,7 +972,9 @@ def _check_children_scope(self):
     @api.multi
     @api.returns('self', lambda value: value.id)
     def copy(self, default=None):
-        default = dict(default or {}, name=_("%s (Copy)") % self.name)
+        default = dict(default or {})
+        if not 'name' in default:
```

Comment:
rather 'name' not in default

Reply:
@oco-odoo comment addressed! Thanks for your review.